test/example/c-api-kitchen-sink.c

```diff
@@ -654,15 +654,27 @@ void test_core() {
     BinaryenSIMDLoad(
       module, BinaryenLoad8x8SVec128(), 0, 8, makeInt32(module, 128)),
     BinaryenSIMDLoad(
-      module, BinaryenLoad8x8UVec128(), 0, 8, makeInt32(module, 128)),
-    BinaryenSIMDLoad(
-      module, BinaryenLoad16x4SVec128(), 0, 8, makeInt32(module, 128)),
-    BinaryenSIMDLoad(
-      module, BinaryenLoad16x4UVec128(), 0, 8, makeInt32(module, 128)),
-    BinaryenSIMDLoad(
-      module, BinaryenLoad32x2SVec128(), 0, 8, makeInt32(module, 128)),
-    BinaryenSIMDLoad(
-      module, BinaryenLoad32x2UVec128(), 0, 8, makeInt32(module, 128)),
+      module, BinaryenLoadExtUVec8x8ToVecI16x8(), 0, 8, makeInt32(module, 128)),
+    BinaryenSIMDLoad(module,
+                     BinaryenLoadExtSVec16x4ToVecI32x4(),
+                     0,
+                     8,
+                     makeInt32(module, 128)),
+    BinaryenSIMDLoad(module,
+                     BinaryenLoadExtUVec16x4ToVecI32x4(),
+                     0,
+                     8,
+                     makeInt32(module, 128)),
+    BinaryenSIMDLoad(module,
+                     BinaryenLoadExtSVec32x2ToVecI64x2(),
+                     0,
+                     8,
+                     makeInt32(module, 128)),
+    BinaryenSIMDLoad(module,
+                     BinaryenLoadExtUVec32x2ToVecI64x2(),
+                     0,
+                     8,
+                     makeInt32(module, 128)),
```

Comment:
I guess this is not related to this PR? (If we want to clang-format this test file, which we didn't end up doing when we mass-clang-formatted the whole codebase, I am in favor, but we can do that on the whole test files in a separate PR.)

Reply:
I think the issue is that on CI we don't check for format on test files, so it allows PRs to land with issues (this fix is for a recent landing). Then when we run clang-format locally it gets fixed up, so I saw it now. But I can remove it from here if you prefer.

Reply:
Unless not fixing this makes the CI fail (it doesn't right?), I think it'd be good to update the whole test files in a separate PR. I was in favor of it when we mass-formated the whole codebase, but I remember you somehow didn't prefer it.

Reply:
It's just that one test file I think - everything else has the proper format. That is, when I run clang-format locally that is the only thing that it decides to change... But, removed from here, I agree it's a distraction.